client/data/media/use-add-media.js

```diff
@@ -29,7 +29,8 @@ export const useAddMedia = () => {
 				addVideopressStatusToFile( file, site ),
 				site,
 				postId,
-				getFileUploader
+				getFileUploader,
```

Comment:
Instead of adding additional arguments and extra complexity, I'd expect us to build a new uploder, a custom version of the `getFileUploader()` function. Then we could pass that to (or expose it through an option of) the `useAddMedia` hook, which would be reused.